tests/test_ipynb.py

```diff
@@ -18,6 +19,8 @@
 from _pytest.monkeypatch import MonkeyPatch
 from tests.util import DATA_DIR
 
+with contextlib.suppress(ModuleNotFoundError):
```

Comment:
This won't work well because line 146 will fail if the module isn't available.

Reply:
on line 25 there's
```
pytest.importorskip("IPython", reason="IPython is an optional dependency")
```
so if `IPython` isn't available, line 146 is never reached